src/settings.c

```diff
@@ -35,8 +35,11 @@
 static GHashTable *settings;
 static FcMonitor *fontconfig_monitor;
 static int fontconfig_serial;
+static ShellIntrospect *shell_introspect;
 static gboolean enable_animations;
 
+static void sync_animations_enabled_override (XdpImplSettings *impl);
```

Comment:
Think we can rename the function to just `sync_animations_enabled()` since it doesn't only use the override from the shell introspection API.